src/components/Comments/Comment.js

```diff
@@ -59,6 +61,7 @@ const Comment = ({
     loadingDm: false,
     loadingDmAnon: false
   });
+  const [userAllowDm, setUserAllowDm] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state variable `userAllowDm` has been introduced. Make sure it's being set and updated correctly throughout the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->